app/static/app.js

```diff
@@ -22,7 +22,11 @@ angular.module('contribute', [
      * leave it on the server and pull it in with a full AJAX URL.
      */
     $routeProvider
-    .when('/examples', {
+    .when('/examples/', {
+        templateUrl: 'examples.html',
+        controller: 'ExamplesController'
+    })
+    .when('/examples/:tech', {
```

Comment:
Is there a way for me to add a optional argument in `ngRoute` without adding a new endpoint?